### PR TITLE
weekly 2022-02-10

### PR DESCRIPTION
**Pull request by pedro-hos**

### CODE REVIEW

**Comment by rpelisse** [comment]
@pedro-hos LGTM! Can you update tomorrow evening, in case there is a couple more news worth mentioning? 

We'll merge on Thursday, so the editorial is up in time.

**Comment by pedro-hos** [comment]
@rpelisse sure, I will update tomorrow w/ some news! No problem :-D

File: src/content/posts/weekly-2022-02-10.adoc [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+link:https://www.keycloak.org/2022/02/adapter-deprecation[Deprecation of Keycloak adapters] by Stian Thorgersen
+
+This blog is a notice that the Keycloak will be deprecating the Keycloak Adapters.
+
+=== Running Arquillian Tests from Eclipse
+
+link:http://www.mastertheboss.com/jboss-frameworks/arquillian/running-arquillian-tests-from-eclipse/[Running Arquillian Tests from Eclipse] by F. Marchioni
+
+The blog guide you how to configure the Arquillian Tests on Eclipse.
```
Comment (oraNod) [inline src/content/posts/weekly-2022-02-10.adoc:50]:
```suggestion
The blog shows you how to configure the Arquillian Tests on Eclipse.
```

File: src/content/posts/weekly-2022-02-10.adoc [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+link:https://blog.ramon-gordillo.dev/2022/02/artemis-monitoring-in-openshift/[Artemis monitoring in OpenShift] by Ramón Gordillo
+
+It is really simple to monitor the brokers deployed on OpenShift and show the metrics in grafana or configuring alerts based on the metrics. We are using the artemis version included in AMQ 7.9 deployed with the operator.
+
+=== Deprecation of Keycloak adapters
+
+link:https://www.keycloak.org/2022/02/adapter-deprecation[Deprecation of Keycloak adapters] by Stian Thorgersen
+
+This blog is a notice that the Keycloak will be deprecating the Keycloak Adapters.
```
Comment (oraNod) [inline src/content/posts/weekly-2022-02-10.adoc:44]:
```suggestion
This blog is a notice that the Keycloak project will be deprecating the Keycloak Adapters.
```

File: src/content/posts/weekly-2022-02-10.adoc [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+link:http://www.mastertheboss.com/jboss-frameworks/arquillian/running-arquillian-tests-from-eclipse/[Running Arquillian Tests from Eclipse] by F. Marchioni
+
+The blog guide you how to configure the Arquillian Tests on Eclipse.
+
+=== Reverse engineer your JBoss AS-WildFly configuration to CLI
+
+link:http://www.mastertheboss.com/jbossas/jboss-script/reverse-engineer-your-jboss-as-wildfly-configuration-to-cli/[Reverse engineer your JBoss AS-WildFly configuration to CLI] by F. Marchioni
+
+Exporting your WildFly/JBoss EAP configuration to a CLI script is something you are going to need one day or another. No worries, the project Profile Cloner comes to rescue!
```
Comment (oraNod) [inline src/content/posts/weekly-2022-02-10.adoc:56]:
```suggestion
Exporting your WildFly/JBoss EAP configuration to a CLI script is something you are going to need one day or another. No worries, the project Profile Cloner comes to the rescue!
```

**Review by oraNod — APPROVED** [review]
Just a few minors for grammar and a suggestion to expand the blurb on the Keycloak Adapters entry. Other than that LGTM. Nice one, Pedro.

**Comment by LightGuard** [comment]
Looking good!

**Pull request merged**